apps/journeys-admin/src/components/Editor/Slider/Settings/Drawer/ImageBlockThumbnail/ImageBlockThumbnail.tsx

```diff
@@ -19,16 +19,34 @@ export function ImageBlockThumbnail({
   Icon = Image3Icon,
   error
 }: ImageBlockThumbnailProps): ReactElement {
-  const isUnsplash =
-    selectedBlock?.src?.match(/^https:\/\/images\.unsplash\.com/) ?? false
+  function getImageSource(srcSet: boolean): string | undefined {
+    const isUnsplash =
+      selectedBlock?.src?.match(/^https:\/\/images\.unsplash\.com/) ?? false
+    const width = '56'
+    const height = '56'
```

Comment:
Can these be made magic numbers of a number type not string?
```suggestion
    const IMG_WIDTH = 56
    const IMG_HEIGHT = 56
```